x-pack/test/ingest_manager_api_integration/apis/epm/install_remove_assets.ts

```diff
@@ -0,0 +1,236 @@
+      });
+      it('should install the pipelines', async function () {
+        if (server.enabled) {
+          const res = await es.transport.request({
+            method: 'GET',
+            path: `/_ingest/pipeline/${logsTemplateName}-${pkgVersion}`,
+          });
+          expect(res.statusCode).equal(200);
+        } else {
+          warnAndSkipTest(this, log);
+        }
+      });
+      it('should install the template components', async function () {
+        if (server.enabled) {
+          const res = await es.transport.request({
+            method: 'GET',
+            path: `/_component_template/${logsTemplateName}-mappings`,
+          });
+          expect(res.statusCode).equal(200);
+          const resSettings = await es.transport.request({
+            method: 'GET',
+            path: `/_component_template/${logsTemplateName}-settings`,
+          });
+          expect(resSettings.statusCode).equal(200);
+        } else {
+          warnAndSkipTest(this, log);
+        }
+      });
+      it('should install the kibana assets', async function () {
+        if (server.enabled) {
+          const resIndexPatternLogs = await kibanaServer.savedObjects.get({
+            type: 'index-pattern',
+            id: 'logs-*',
+          });
+          expect(resIndexPatternLogs.id).equal('logs-*');
+          const resIndexPatternMetrics = await kibanaServer.savedObjects.get({
+            type: 'index-pattern',
+            id: 'metrics-*',
+          });
+          expect(resIndexPatternMetrics.id).equal('metrics-*');
+          const resIndexPatternEvents = await kibanaServer.savedObjects.get({
+            type: 'index-pattern',
+            id: 'events-*',
+          });
+          expect(resIndexPatternEvents.id).equal('events-*');
+          const resDashboard = await kibanaServer.savedObjects.get({
+            type: 'dashboard',
+            id: 'sample_dashboard',
+          });
+          expect(resDashboard.id).equal('sample_dashboard');
+          const resDashboard2 = await kibanaServer.savedObjects.get({
+            type: 'dashboard',
+            id: 'sample_dashboard2',
+          });
+          expect(resDashboard2.id).equal('sample_dashboard2');
+          const resVis = await kibanaServer.savedObjects.get({
+            type: 'visualization',
+            id: 'sample_visualization',
+          });
+          expect(resVis.id).equal('sample_visualization');
+          const resSearch = await kibanaServer.savedObjects.get({
+            type: 'search',
+            id: 'sample_search',
+          });
+          expect(resSearch.id).equal('sample_search');
+        } else {
+          warnAndSkipTest(this, log);
+        }
+      });
+    });
+
+    describe('uninstalls all assets when uninstalling a package', async () => {
+      before(async () => {
+        if (server.enabled) {
+          await deletePackage(pkgKey);
+        }
+      });
+      it('should uninstall the index templates', async function () {
```

Comment:
Same as for the installation case, I would call these "should have uninstalled..." instead of "should uninstall"